bindings/python/Makefile.am

```diff
@@ -33,7 +33,7 @@ EXTRA_DIST = \
 if ENABLE_PYTHON_BINDING
 
 all-local:
-	$(PYTHON) $(srcdir)/setup.py build
+	CXXFLAGS="$(CXXFLAGS)" $(PYTHON) $(srcdir)/setup.py build
```

Comment:
I was under the impression that the python dist-utils already took care of specifying ``-std=c++11``. Would it still break without this change?